STATUS: INLINE FAIL
MODIFIED: com.j256.ormlite.dao.RuntimeExceptionDaoTest.testCloseLastIteratorThrows
MODIFIED FILE: /home/alex/Repositories/ormlite-core/src/test/java/com/j256/ormlite/dao/RuntimeExceptionDaoTest.java
CLEANER: com.j256.ormlite.logger.LoggerFactoryTest.testLogFactoryProperty
CLEANER FILE: /home/alex/Repositories/ormlite-core/src/test/java/com/j256/ormlite/logger/LoggerFactoryTest.java
POLLUTER: com.j256.ormlite.logger.LoggerFactoryTest.testSetLogFactory
POLLUTER FILE: /home/alex/Repositories/ormlite-core/src/test/java/com/j256/ormlite/logger/LoggerFactoryTest.java
ORIGINAL CLEANER SIZE: 7
NEW CLEANER SIZE: 1
ELAPSED TIME: 19.325000047683716
==========================
@@ -1137,0 +1137,3 @@
+ {
+     LoggerFactory.setLogBackendFactory(null);
+ }
